### PR TITLE
Add rubocop-rake

### PR DESCRIPTION
**Pull request by tejasbubane**
https://github.com/rubocop/rubocop-rake

-----------------

Before submitting the PR make sure the following are checked:

* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Wrote [good commit messages][1].
* [ ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [x] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [ ] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by tejasbubane** [comment]
Rubocop failures are unrelated and shall be fixed in https://github.com/rubocop/rubocop-rspec/pull/1154

**Comment by bbatsov** [comment]
Looks good, but you'll have to fix the broken build.

**Comment by bbatsov** [comment]
@tejasbubane ping :-)

**Comment by tejasbubane** [comment]
Waiting for https://github.com/rubocop/rubocop-rake/pull/37 to be merged.

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ gem 'rspec', '~> 3.7'
 # RuboCop Performance upgrade to 1.11 is postponed until
 # https://github.com/rubocop/rubocop/pull/9721 will be resolved.
 gem 'rubocop-performance', '~> 1.10.0'
+gem 'rubocop-rake', '~> 0.5.1'
```
Comment (koic) [inline Gemfile:14]: RuboCop Rake 0.6.0 has been released. Can you update to 0.6.0?
https://github.com/rubocop/rubocop-rake/releases/tag/v0.6.0

```suggestion
gem 'rubocop-rake', '~> 0.6.0'
```

File: tasks/prof.rake [hunk 2]
```diff
@@ -10,6 +10,7 @@ namespace :prof do
     system cmd
   end
 
+  desc 'Run Rubocop on itself only if dump does not exist'
```
Comment (koic) [inline tasks/prof.rake:13]:
```suggestion
  desc 'Run RuboCop on itself only if dump does not exist'
```

**Pull request merged**